docs/guides/docs/running-a-node/running-a-local-node.mdx

```diff
@@ -37,71 +37,36 @@ Or to deploy without using a signing key:
 forc deploy --unsigned --node-url 127.0.0.1:4000/graphql
 ```
 
-## Local node (with state persistence)
+### Funding a wallet locally
```

Comment:
The chain configuration not only funds wallets locally but can also manipulate many more aspects. Can we keep the title as 'Chain Configuration' (same as the beta-4 guide), but include additional text to clarify that this is the method for locally funding a wallet immediately?